Cargo.toml

```diff
@@ -1,8 +1,22 @@
 [package]
 name = "starship"
-version = "0.1.0"
-authors = ["Matan Kushner <hello@matchai.me>"]
+version = "0.1.0-beta.4"
 edition = "2018"
```

Comment:
2018? Not 2019?

Reply:
Yep. Rust edition 2018 was released at the end of last year and is still the latest edition.

Reply:
I see @matchai